src/components/configuration.rs

```diff
@@ -88,7 +88,7 @@ impl Config {
             )
             .set_override_option("server.host", args.host)?
             .set_override_option("server.port", args.port)?
-            .set_default("synapse.url", "https://synapse.decentraland.zone")?
+            .set_default("synapse.url", "https://synapse.decentraland.org")?
```

Comment:
is this intentional? I mean do we want to use prod for local development?

Reply:
I used it for testing, great catch!